posixlib/src/main/resources/scala-native/sys/socket.c

```diff
@@ -1,10 +1,11 @@
 #include <string.h>
+#include <stddef.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <errno.h>
-#include "../netinet/in.h"
+// #include "../netinet/in.h"
```

Comment:
I think this should be removed.

Reply:
Thank you for catching.  Removed in 2 places.  Successfully building.